packages/full-stack-tests/src/hierarchy-builder/grouping/AutoExpand.test.ts

```diff
@@ -311,5 +311,97 @@ describe("Stateless hierarchy builder", () => {
         });
       });
     });
+
+    describe("Properties grouping", () => {
+      const propertiesAutoExpandAlways: ECSqlSelectClauseGroupingParams = {
+        byProperties: {
+          fullClassName: "BisCore.Element",
+          propertyGroups: [{ propertyName: "UserLabel", propertyValue: "test1" }],
+          autoExpand: "always",
+        },
+      };
+
+      const propertiesAutoExpandSingleChild: ECSqlSelectClauseGroupingParams = {
+        byProperties: {
+          fullClassName: "BisCore.Element",
+          propertyGroups: [{ propertyName: "UserLabel", propertyValue: "test1" }],
+          autoExpand: "always",
```

Comment:
This is how we check if autoExpand is the same: 
![image](https://github.com/iTwin/presentation/assets/100586436/47e02666-0fc9-4886-a17a-7ec29cdc0ae9)
Since autoExpand is undefined, this is never checked. Changed to check if it is false:
![image](https://github.com/iTwin/presentation/assets/100586436/8c29f912-4f2c-484b-bad7-cb79688c82ca)
Did the same in other tests in this file.